 std_msgs/Empty request
 ---
 hand_eye_calibration/CameraPoseNodeInfo info